pkg/standalone/uninstall.go

```diff
@@ -98,6 +98,9 @@ func Uninstall(uninstallAll bool, dockerNetwork string, containerRuntime string,
 	if err != nil {
 		print.WarningStatusEvent(os.Stdout, "WARNING: could not delete dapr bin dir: %s", daprBinDir)
 	}
+	// We don't delete .dapr/scheduler by choice since it holds state.
```

Comment:
design decision: decided to document this decision in code to avoid someone from adding it without a discussion. Basically: do we want to delete the data on the regular uninstall (without `--all`)? I don't think so because we also don't remove the components folder or redis. The `uninstall` command does not destroy any data as of today, only with `--all`. So, I am document this decision here.